docs/AllelicCapSeg/ACS-methods.tex

```diff
@@ -109,7 +109,7 @@ \subsection{Segmented model} \label{segmented-model}
 \begin{equation}
 P(z_{ja(r,o)} = 1 | {\bf \pi}) = \pi_{a(r,o)}
 \end{equation}
-We set the alt and ref minor probabilities equal so that the only free parameter is $\pi_o$, with $\pi_{a(r)} = (1 - \pi_o)/2$.
+We set the alt and ref minor probabilities equal so that the only free parameter is $\pi = \pi_o$, with $\pi_{a(r)} = (1 - \pi)/2$.
 %
 The Bayesian network corresponding to this model is shown in Figure \ref{graphical_model}.
```

Comment:
speaking of which, can we make a similar figure for the copy-ratio model?  i might have already asked, but did you use a particular program/package to generate this?